### ISSUE TITLE
Blank lines below switch cases without scopes aren't indented

### ISSUE THREAD

**Issue by AdamNiederer** [orzechowskid/tsi.el#21]
All cases tested with tsi-typescript-indent-offset = 2

Given this case:

```js
switch(0) {
  case 0:

}
```

Moving the cursor to the third (blank) line and indenting yields a zero-space indent, like so (`|` representing the cursor):

```js
switch(0) {
  case 0:
|
}
```

When that third line is not blank, it will indent to the correct level (in this case 4)
```js
switch(0) {
  case 0:
    0
}
```

This can make it a bit awkward to write switch cases, although electric indent can make this a bit less cumbersome. For reference, js-mode will indent the first example like so:

```js
switch(0) {
  case 0:
    |
}
```

**Comment by stevemolitor** [comment]
This is fixed by #18.

**Comment by AdamNiederer** [comment]
Confirmed working; thank you!
